### PR TITLE
Ppi may23

### PR DESCRIPTION
**Pull request by MSalavei**

### CODE REVIEW

File: PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- Update two concepts and set for 2 concepts diffirent concept_class_id
```
Comment (TinyRickC137) [inline PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql:1]: Typo here. The correct one would sound like this:
Update two concepts and set different concept_class_id for 2 concepts

**Review by hardhouse — DISMISSED** [review]

**Review by Alexdavv — APPROVED** [review]

File: PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        EXECUTE format('create table %I as select * from concept_relationship_manual',
+                       'concept_relationship_manual_backup_' || update);
+    END
+$body$;
+
+
+
+-- Update 3 concepts and set different concept_class_id for 2 concepts
+
+--Deprecating existing mappings and making two question concepts standard
```
Comment (Alexdavv) [inline PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql:33]: We can easily insert the content directly to the manual tables and publish the respective csvs

Comment (Alexdavv) [reply PPI/manual_work/May_2023/PPI_concepts_correction_May2023.sql:33]: And we need to organize the manual tables as the full set

**Review by Alexdavv — APPROVED** [review]

**Review by TinyRickC137 — APPROVED** [review]
Manual changes approved. 
Comments transferred into TODO section; new task created (AVOF-3982)

**Review by hardhouse — APPROVED** [review]

**Pull request merged**